src/Data/Maybe.agda

```diff
@@ -56,6 +57,29 @@ monadPlus {f} = record
   nothing ∣ y = y
   just x  ∣ y = just x
 
+
+------------------------------------------------------------------------
+-- Other associated operations
+
+fromMaybe : {ℓ : Level} {A : Set ℓ} (default : A) (try : Maybe A) → A
+fromMaybe def (just x) = x
+fromMaybe def nothing  = def
+
+listToMaybe : {ℓ : Level} {A : Set ℓ} → List A → Maybe A
+listToMaybe []       = nothing
+listToMaybe (x ∷ xs) = just x
+
```

Comment:
Note that this is in #354 and it is called `head` which I think makes its behaviour clearer.
`to`/`from` functions should in my opinion be only be used for injective functions.